features/support/document_helper.rb

```diff
@@ -13,6 +13,14 @@ module DocumentHelper
 
   SEARCH_ENDPOINT = "#{Plek.current.find('search')}/search.json".freeze
 
+  def stub_business_finder_qa_config
+    allow_any_instance_of(QaController).to receive(:qa_config).and_return(business_finder_qa_config)
+  end
+
+  def business_finder_qa_config
+    @business_finder_qa_config ||= YAML.load_file('./features/fixtures/prepare_business_uk_leaving_eu.yaml')
+  end
```

Comment:
Is this additional code necessary or can we call the equivalent code in qa_helper.rb?

Reply:
Perhaps they could be consolidated in [FixturesHelper](https://github.com/alphagov/finder-frontend/blob/66270f78f02810a084705bae886a1f3bcf28644c/spec/support/fixtures_helper.rb)?